Ang user functionality

LovetheFrogs opened the pull request:
Branch DOES NOT WORK ON ITS OWN!!, as it needs changes only present on branch #155

xIvqn committed:
feat: app logo

anna-trofimova committed:
Merge branch 'dev' of https://github.com/CodeURJC-DAW-2022-23/webapp9 into dev

anna-trofimova committed:
Create:angular project

anna-trofimova committed:
Added: models classes

anna-trofimova committed:
Added: home page

It is needed to change css properties

anna-trofimova committed:
Added: home destinations

anna-trofimova committed:
Update

anna-trofimova committed:
Added: api rest funcionando

anna-trofimova committed:
Fixed:routing

anna-trofimova committed:
Added: logIn functionality

anna-trofimova committed:
Added:home page elements

anna-trofimova committed:
update:is logged function fixed

anna-trofimova committed:
Fixed: LogIn works

anna-trofimova committed:
Fixed: image links in home page

anna-trofimova committed:
Added: signUp function

anna-trofimova committed:
Fixed: isAdmin function and user`s pic

anna-trofimova committed:
Added: chart most visited destinations

xIvqn committed:
Fix: Send registration emails API

anna-trofimova committed:
Fixed: LogIn CSS

anna-trofimova committed:
Fixed: carousel item

anna-trofimova committed:
Fixed: css home page

anna-trofimova committed:
Fixed: css sign up page

anna-trofimova committed:
Update sign-up.component.ts

xIvqn committed:
Fix: Docker compose typo

xIvqn committed:
Fix: Details buttons in home

xIvqn committed:
Fix: Navbar search link and query

Co-authored-by: Adaya:) <103607990+AdayaUwU@users.noreply.github.com>

xIvqn committed:
Fix: Views label on chart

xIvqn committed:
Merge pull request #152 from CodeURJC-DAW-2022-23/ang-home-page

Ang home page

LovetheFrogs committed:
Feature: Added myItineraries page

LovetheFrogs committed:
Feature: Added myItineraries component

LovetheFrogs committed:
Feature: Added profile page

LovetheFrogs committed:
Feature: Added profile component

LovetheFrogs committed:
Feature: Updated Itineraries service

Added new methods to check API

LovetheFrogs committed:
Feature: Added user service

AdayaUwU reviewed (changes requested):
There is no point on merging changes that we can't even check if they work, and you haven't even bothered to change the app routing file to make the urls work. This PR won't be accepted until the changes can be properly tested.

LovetheFrogs committed:
Fix: Moved Extraction of itineraries to new method

This aims to achieve page reloading correctly all of the times after data changes
